src/git-extract

```diff
@@ -58,9 +58,9 @@ function preconditions {
 
 function steps {
```

Comment:
Hmm, what this method does is defining the steps that this command executes. `define_steps` is a bit long. So calling it `steps`, meaning this method defines and returns the steps that this command runs, seems appropriate here. 

If there is a specifier like `add` in `add_steps`, it feels like there should also be a `remove_steps`, a `run_steps` etc in this API.